PWGHF/TableProducer/treeCreatorToXiPi.cxx

```diff
@@ -0,0 +1,327 @@
+} // namespace full
+
+DECLARE_SOA_TABLE(HfToXiPiFull, "AOD", "HFTOXIPIFULL",
+                  full::XPv, full::YPv, full::ZPv, collision::NumContrib,
+                  full::XDecayVtxOmegac, full::YDecayVtxOmegac, full::ZDecayVtxOmegac,
+                  full::XDecayVtxCascade, full::YDecayVtxCascade, full::ZDecayVtxCascade,
+                  full::XDecayVtxV0, full::YDecayVtxV0, full::ZDecayVtxV0,
+                  full::SignDecay, full::Chi2PCAOmegac,
+                  full::CovVtxOmegacXX, full::CovVtxOmegacYY, full::CovVtxOmegacZZ,
+                  full::CovVtxCascadeXX, full::CovVtxCascadeYY, full::CovVtxCascadeZZ,
+                  full::CovVtxV0XX, full::CovVtxV0YY, full::CovVtxV0ZZ,
+                  full::PxOmegac, full::PyOmegac, full::PzOmegac,
+                  full::PxCasc, full::PyCasc, full::PzCasc,
+                  full::PxPrimaryPi, full::PyPrimaryPi, full::PzPrimaryPi,
+                  full::PxLambda, full::PyLambda, full::PzLambda,
+                  full::PxPiFromCasc, full::PyPiFromCasc, full::PzPiFromCasc,
+                  full::PxPosV0Dau, full::PyPosV0Dau, full::PzPosV0Dau,
+                  full::PxNegV0Dau, full::PyNegV0Dau, full::PzNegV0Dau,
+                  full::ImpactParCascXY, full::ImpactParPrimaryPiXY, full::ImpactParV0XY, full::ImpactParOmegacXY,
+                  full::ImpactParCascZ, full::ImpactParPrimaryPiZ, full::ImpactParV0Z, full::ImpactParOmegacZ,
+                  full::InvMassLambda, full::InvMassCascade, full::InvMassOmegac,
+                  full::CosPAV0, full::CosPAOmegac, full::CosPACasc, full::CosPAXYV0, full::CosPAXYOmegac, full::CosPAXYCasc,
+                  full::CTauOmegac, full::CTauCascade, full::CTauV0, full::CTauXic,
+                  full::EtaV0PosDau, full::EtaV0NegDau, full::EtaPiFromCasc, full::EtaPiFromOme,
+                  full::EtaOmegac, full::EtaCascade, full::EtaV0,
+                  full::DcaXYToPvV0Dau0, full::DcaXYToPvV0Dau1, full::DcaXYToPvCascDau,
+                  full::DcaZToPvV0Dau0, full::DcaZToPvV0Dau1, full::DcaZToPvCascDau,
+                  full::DcaCascDau, full::DcaV0Dau, full::DcaOmegacDau,
+                  full::StatusPidLambda, full::StatusPidCascade, full::StatusPidOmegac,
+                  full::StatusInvMassLambda, full::StatusInvMassCascade, full::StatusInvMassOmegac, full::ResultSelections,
+                  full::FlagMcMatchRec, full::DebugMcRec);
+
+DECLARE_SOA_TABLE(HfToXiPiEvents, "AOD", "HFTOXIPIEVENTS",
+                  collision::NumContrib,
+                  collision::PosX,
+                  collision::PosY,
+                  collision::PosZ);
+} // namespace o2::aod
+
+/// Writes the full information in an output TTree
+struct HfTreeCreatorToXiPi {
+
+  Produces<o2::aod::HfToXiPiFull> rowCandidateFull;
+  Produces<o2::aod::HfToXiPiEvents> rowCandidateEvents;
+
+  Configurable<bool> doNotMc{"doNotMc", true, "Process data tree writer"};
+  Configurable<bool> doMc{"doMc", false, "Process MC tree writer"};
```

Comment:
These are not needed. Each process function has its own configurable.

Reply:
Ok